### ISSUE TITLE
ImportError: cannot import name tensorboard_server 

### ISSUE THREAD

**Issue by aabobakr** [tensorflow/tensorflow#1281]
I've successfully built tensorflow "version: 0.6.0 " and tensor board from source on Mac OS X. The problem is I am getting this error when I try to run the tensorboard module. 

command: 
`
./bazel-bin/tensorflow/tensorboard/tensorboard --help
`

the produced error 

```
Ahmeds-MacBook-Pro:tensorflow ahmedabobakr$ ./bazel-bin/tensorflow/tensorboard/tensorboard --help
Traceback (most recent call last):
  File "/Users/ahmedabobakr/tensorflow/bazel-bin/tensorflow/tensorboard/tensorboard.runfiles/tensorflow/tensorboard/backend/tensorboard.py", line 36, in <module>
    from tensorflow.tensorboard.backend import tensorboard_server
ImportError: cannot import name tensorboard_server

```

**Comment by vrv** [comment]
We would strongly suggest upgrading to 0.7.1 (sync to the 0.7.1 tag if needed, or HEAD if you are daring). tensorboard had a few bugs in 0.6.0 which should mostly be fixed now.

**Comment by girving** [comment]
Closing due to lack of activity.  Please reopen if it's not fixed in 0.7.1.
